packages/broker/src/broker.ts

```diff
@@ -84,6 +84,9 @@ export const createBroker = async (config: Config): Promise<Broker> => {
     const storageNodeRegistry = StorageNodeRegistry.createInstance(config, storageNodes)
 
     const usePredeterminedNetworkId = !config.generateSessionId || config.plugins['storage']
+
+    const webrtcDisallowPrivateAddresses = config.network?.webrtcDisallowPrivateAddresses || false
```

Comment:
From what I understand (@teogeb can correct me if I'm wrong) you can rely on the default value of the schema here being present

```suggestion
    const webrtcDisallowPrivateAddresses = config.network.webrtcDisallowPrivateAddresses
```